src/appleseed/renderer/kernel/intersection/intersector.h

```diff
@@ -116,6 +116,22 @@ class Intersector
         const size_t                        primitive_index,
         const TriangleSupportPlaneType&     triangle_support_plane) const;
 
+    // Manufacture a procedural surface hit "by hand".

```

Comment:
I can't comment there but please update the comment for the `make_triangle_shading_point()` method above (it should at least mention the word "triangle").